plugins/utilities/share_replay.py

```diff
@@ -0,0 +1,192 @@
+# ba_meta require api 7
+from __future__ import annotations
+from typing import TYPE_CHECKING, cast
+if TYPE_CHECKING:
+    from typing import Any, Sequence, Callable, List, Dict, Tuple, Optional, Union
+
+from os import listdir, mkdir
+from shutil import copy, copytree
+import ba
+import _ba
+from bastd.ui.watch import WatchWindow as ww
+from bastd.ui.popup import PopupWindow
+
+# mod by ʟօʊքɢǟʀօʊ
+# export replays to mods folder and share with your friends or have a backup
+
+
+def Print(*args, color=None, top=None):
+    out = ""
+    for arg in args:
+        a = str(arg)
+        out += a
+    ba.screenmessage(out, color=color, top=top)
+
+
+def cprint(*args):
+    out = ""
+    for arg in args:
+        a = str(arg)
+        out += a
+    _ba.chatmessage(out)
+
+
+internal_dir = "/data/data/net.froemling.bombsquad/files/bombsquad_config/replays"
+external_dir = "/storage/emulated/0/Android/data/net.froemling.bombsquad/files/mods/replays"
```

Comment:
Have to fix this

Reply:
`_ba.env()` might help

Reply:
Ok will look at that,thanks

Reply:
`ba.modutils._request_storage_permission()` can i request permission and copy files to the downloads folder?

Reply:
Will the permission be revoked after the app restarts or Sumthing??

Reply:
Because i don't want to expose user's internal storage to potential malware

Reply:
haven't tried it before but it sure seems interesting!

Reply:
Maybe ill have an option to either copy to the directory with write perms or take the risk and copy to somewhere easily accessible